assignment/week01/assign3/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"/>
+    <title>유진's 사진관</title>
+</head>
+<body>
+    <header id="title">2023 여름 서유럽 여행기</header>
+    <section id="preview">
+        <img src="img/bel/1.jpg" alt="벨기에 사진 1"/>
```

Comment:
img alt 부분에는 띄어쓰기가 있으면 좋지 않다구 합니다!
alt 태그가 없으면 스크린리더가 src 파일명을 읽어주는것처럼, alt 태그에 띄어쓰기가 있어도 src 파일명을 읽기 때문에 접근성 측면에서 지양된다고 해용

```suggestion
        <img src="img/bel/1.jpg" alt="벨기에_사진_1"/>
```
처럼 _로 이어주는게 좋을것 같습니댜